### ISSUE TITLE
Failed auth: Client/server version mismatch

### ISSUE THREAD

**Issue by chmanie** [muesli/obs-cli#61]
I'm trying to use this tool (v0.5.0) with OBS v27.2.4 and obs-websocket v5.0.0.

When trying to do `obs-cli scene list` I'm getting the following error message:
```
error: Failed auth: Client/server version mismatch? Unrecognized message: {"d":{"obsWebSocketVersion":"5.0.0","rpcVersion":1},"op":0}
```

I disabled authentication for this test.

What am I missing?

EDIT: I just saw that v5.0.0 of obs-websocket was just released 3 days ago. Well, that must be why :) Let me know if I can help in any way.

**Comment by fdaciuk** [comment]
Yeah, same problem here because obs-websocket just updated from v4.9.1 to v5.0.0 =)

**Comment by v3eil** [comment]
Same

> error: Failed auth: Client/server version mismatch? Unrecognized message: {"d":{"obsWebSocketVersion":"5.0.1","rpcVersion":1},"op":0}

**Comment by genpfault** [comment]
[`obs-websocket-4.9.1-compat`](https://github.com/obsproject/obs-websocket/releases/tag/4.9.1-compat) Worked For Me™ on Windows.

**Comment by mystiquewolf** [comment]
> The [`obs-websocket-4.9.1-compat`](https://github.com/obsproject/obs-websocket/releases/tag/4.9.1-compat) plugin Worked For Me™ with OBS 28.0.1 on Windows.

Unfortuntely doesn't seem to work on Kubuntu 23.04

**Comment by profdecoy** [comment]
> > The [`obs-websocket-4.9.1-compat`](https://github.com/obsproject/obs-websocket/releases/tag/4.9.1-compat) plugin Worked For Me™ with OBS 28.0.1 on Windows.
> 
> Unfortuntely doesn't seem to work on Kubuntu 23.04

If you use the PPA from OBS themselves, the compat websocket package does work.  However, you have to force the version from the PPA and apt flags it as having broken dependencies.

**Comment by grigio** [comment]
```
obs-cli recording toggle
error: Failed auth: Client/server version mismatch? Unrecognized message: {"d":{"obsWebSocketVersion":"5.2.2","rpcVersion":1},"op":0}
```

**Comment by mccollam** [comment]
I got the package working on Ubuntu, but had to symlink the libraries to the appropriate place before OBS Studio would pick them up.  The process is described here: https://github.com/obsproject/obs-websocket/issues/995

**Comment by grigio** [comment]
https://github.com/grigio/obs-cmd

if you are interested it's a minimal implementation of obs websocket 5

**Comment by pschmitt** [comment]
I couldn't find any alternative to obs-cli that supports scene switching, items, filters and hotkeys that works with the new websockets API.

So here's my implementation, in Python:
https://github.com/pschmitt/obs-cli

**Comment by poleguy** [comment]
I wish pschmitt's version showed up on the google main search page. It actually works without a bunch of hoop jumping for a simple stop veritual cam cli.
